paasta_tools/cli/cmds/secret.py

```diff
@@ -211,9 +220,36 @@ def paasta_secret(args):
             sys.exit(1)
     else:
         service = args.service
+
+    # Check if "decrypt" and "secrets_for_teams" first to avoid vault auth
+    if args.action == "decrypt":
+        secrets_for_teams_only = is_secrets_for_teams_enabled(
+            service, args.yelpsoa_config_root
+        )
+        if secrets_for_teams_only:
+            clusters = (
+                args.clusters.split(",")
+                if args.clusters
+                else list_clusters(service=service, soa_dir=os.getcwd())
+            )
+
+            if len(clusters) > 1 or not clusters:
+                print(
+                    "Can only decrypt for one specified cluster at a time!\nFor example,"
+                    " try '-c norcal-devc' to decrypt the secret for this service in norcal-devc."
+                )
+                sys.exit(1)
+
+            print(
+                get_kubernetes_secret(args.secret_name, service, clusters),
+                end="",
```

Comment:
I think it's alright the have the trailing newline, this is not really meant to be called from other scripts anyway. And avoids the issue for having the next bash prompt on the same line of the output.